native/screens/Applicants/Applicants.jsx

```diff
@@ -87,21 +81,56 @@ const RoleList = ({ applicants, navigation, role, currProject }) => {
             }}
           >
             <View>
-              <View>
+              <View style={styles.applicantInfo}>
                 <Text>IMAGE</Text>
+                <Text
+                  style={styles.boldText}
+                >{`${applicant.first_name} ${applicant.last_name}`}</Text>
+                <Text style={styles.boldText}>{`${applicant.role}`}</Text>
               </View>
-              <Text>{`${applicant.first_name} ${applicant.last_name}`}</Text>
-              <Text>{applicant.role}</Text>
-              <Text>PERSONAL MESSAGE FROM APPLICANT</Text>
+              <Text
+                style={styles.applicantMessage}
+              >{`here is a personal message from ${applicant.first_name} about their application for this project and expressing their interest`}</Text>
```

Comment:
so weird that I'm checked into your branch as I'm reading this PR and this is not what I see on line 91 lol
What I see: 
`              <Text style={styles.applicantMessage}>PERSONAL MESSAGE FROM APPLICANT</Text>
`
Wonder why GitHub is showing me something different